core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java

```diff
@@ -63,9 +58,9 @@ private CachableBlockFile() {}
 
   private static final Logger log = LoggerFactory.getLogger(CachableBlockFile.class);
 
-  public static class Writer implements BlockFileWriter {
```

Comment:
Seems like this whole writer class could be refactored away. It does nothing related to caching and is just a thing wrapper for BCfile.  Could do this in another PR

Reply:
Yeah I am hoping these changes open up more areas of improvement.  I can do this next.